datastore/writer/flask_frontend/routes_handler.py

```diff
@@ -37,6 +41,24 @@ def reserve_ids():
     return JsonResponse({"ids": ids})
 
 
+@handle_internal_errors
+def write_action_worker():
+    if not request.is_json:
+        raise InvalidRequest("Data must be json")
+    if len(cast(Dict[str, Any], request.json).get("events", ())) != 1:
+        raise InvalidRequest("write_action_worker may contain only 1 event!")
+    if collection_from_fqid(request.json["events"][0]["fqid"]) != "action_worker":  # type: ignore
+        raise InvalidRequest("Collection for write_action_worker must be action_worker")
+    write_handler = WriteHandler()
+    write_handler.write_action_worker(request.json)
+    return (
+        "",
+        201
+        if cast(Dict[str, Any], request.json)["events"][0]["type"] == "create"
+        else 200,
```

Comment:
The `write` method always returns a 201, since it is always about creating some events and positions. If you want to be consistent, we should do the same here, or do we have any advantage in returning a 200?

Reply:
Done